test/graphql/queries/system_query_test.rb

```diff
@@ -66,7 +66,7 @@ class SystemQueryTest < ActiveSupport::TestCase
         :test_result,
         profile: @profile1,
         host: @host1,
-        end_time: 1.minute.ago,
+        end_time: 1.day.ago,
```

Comment:
This seems unnecessary? `latest` should work, even if the times differ by seconds.

Reply:
Sorry :see_no_evil: I just left this here by accident